addon-test-support/build-owner.js

```diff
@@ -42,11 +39,7 @@ export default function(resolver = getResolver()) {
   function register(name, factory) {
     let thingToRegisterWith = registry || container;
 
-    if (
-      !(container.factoryFor
-        ? container.factoryFor(name)
-        : container.lookupFactory(name))
-    ) {
+    if (!(container.factoryFor ? container.factoryFor(name) : container.lookupFactory(name))) {
```

Comment:
maybe we should extract an `existingFactory` variable here to make the condition easier to read